search-commons/src/test/java/no/unit/nva/search/resource/ResourceClientTest.java

```diff
@@ -529,16 +533,21 @@ void shouldReturnCaseInsensitiveCourses(String searchValue, int expectedHits)
         assertEquals(expectedHits, pagedSearchResourceDto.totalHits());
     }
 
-    @Test
-    void isSearchingForAllPublicationsDoWork() throws UnauthorizedException, BadRequestException {
+    @ParameterizedTest
+    @CsvSource({
+        "5,MANAGE_RESOURCES_ALL",
+        "22,MANAGE_CUSTOMERS",
+        "22,MANAGE_CUSTOMERS,MANAGE_RESOURCES_ALL"
+    })
+    void isSearchingForAllPublicationsAsRoleWork(int expectedHits, String accessRights)
+            throws UnauthorizedException, BadRequestException {
+        var rights = Arrays.stream(accessRights.split(COMMA)).map(AccessRight::valueOf).toList();
```

Comment:
Would it work to replace the `@CsvSource` with a `@MethodSource` that returns a list, instead of parsing the input string?